app/models/case_log.rb

```diff
@@ -47,6 +48,60 @@ class CaseLog < ApplicationRecord
 
   enum status: { "not_started" => 0, "in_progress" => 1, "completed" => 2 }
 
+  enum ethnic: DbEnums.ethnic
```

Comment:
Could we put the enums into the module as well or do they have to be here?

Reply:
I don't think we can put it in module, it's something to do with being part of ActiveModel